src/components/popup/home/Balance.tsx

```diff
@@ -111,18 +111,9 @@ export default function Balance() {
     })();
   }, [activeAddress]);
 
-  // router push
-  const [push] = useHistory();
-
   // display theme
   const theme = useTheme();
 
-  // lock wallet and terminate session
-  async function lockWallet() {
-    await removeDecryptionKey();
-    push("/unlock");
```

Comment:
No need for a route change anymore. Removing the decryption key makes the root re-render automatically, showing the Unlock screen.